src/components/teamPhotos.js

```diff
@@ -1,6 +1,11 @@
 import React from "react"
 import Img from "gatsby-image"
 import styled from "styled-components"
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons"
+import { faLinkedinIn } from "@fortawesome/free-brands-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { SB_NAVY } from "@colors"
+import MediaQuery from "react-responsive"
```

Comment:
Because with `StyledIcon` we're passing color as a prop to the component. Using `@media`, I didn't see a way to only pass white for `999px` and below and only pass sb_navy for `1000px`+. I could change the code so that the css looks like this 
`const StyledIcon = styled(FontAwesomeIcon)`
  color: "white";
  @media (max-width: 999px) {
    color: ${SB_NAVY}
  }`
but then there wouldn't really be a point of passing the color prop to `StyledIcon`. This way, we can still keep it as a function that accepts a color. What do you think?